src/agent.ts

```diff
@@ -0,0 +1,54 @@
+import { Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+import { ethers } from "ethers";
+import { createFinding } from "./findings";
+import {
+  NETHERMIND_DEPLOYER_ADDRESS,
+  AGENT_REGISTRY_ADDRESS,
+  CREATE_AGENT_ABI,
+  UPDATE_AGENT_ABI,
+  DISABLE_AGENT_ABI,
+} from "./constants";
+
+const provideHandleTransaction = (
+  deployerAddress: string,
+  agentRegistryAddress: string,
+  createAgentAbi: string,
+  updateAgentAbi: string,
+  disableAgentAbi: string
+): HandleTransaction => {
+  const iface = new ethers.utils.Interface([
```

Comment:
Unless I'm missing it, `iface` doesn't seem to be used. If so, it can be removed.